src/KBarPortal.tsx

```diff
@@ -5,6 +5,7 @@ import { useKBar } from "./useKBar";
 
 interface Props {
   children: React.ReactNode;
+  host?: HTMLElement;
```

Comment:
Let's just use the same name as what's passed through to Portal:

```suggestion
  container?: HTMLElement;
```

Reply:
sure, done. 

Let me know if extending KBarPortal Prop with Radix Portal is a good idea. 

<img width="672" alt="Screenshot 2023-02-20 at 11 08 35 PM" src="https://user-images.githubusercontent.com/22031931/220171395-e94ad77e-9c7f-4db7-834a-2aaa3b3b944b.png">

Reply:
sure, done.

Let me know if extending KBarPortal Prop with Radix Portal is a good idea.

<img width="572" alt="Screenshot 2023-02-20 at 11 09 47 PM" src="https://user-images.githubusercontent.com/22031931/220171575-a87e4b23-857e-4806-88b4-b07c71ea81d1.png">

Reply:
PortalProps includes `children` by default yea?

Reply:
Yup. it has default `children` prop.

Reply:
so we can just do:

```
export function `KBarPortal(props: PortalProps) {
  // ...
}
```

Reply:
yes we can. should i make the changes accordingly ?